examples/research_projects/xtreme-s/README.md

```diff
@@ -126,10 +125,9 @@ The following command shows how to fine-tune the [XLS-R](https://huggingface.co/
 python -m torch.distributed.launch \
     --nproc_per_node=2 \
     run_xtreme_s.py \
+    --task="minds14" \
+    --language="all" \
     --model_name_or_path="facebook/wav2vec2-xls-r-300m" \
-    --dataset_name="google/xtreme_s" \
-    --dataset_config_name="minds14.all" \
-    --eval_split_name="test" \
```

Comment:
Eval should default to "validation" and I've added a new model "prediction" which should only be run in the end. It's important that we use "validation" during training and "test" after training (won't change) your results here